### ISSUE TITLE
Is there a route from a vis.js graph back to a JSON file?

### ISSUE THREAD

**Issue by wolfig** [almende/vis#1721]
Hi Experts,

this is not an issue (well for me it is...), but rather a question for advice.
I'm using vis.js to visualize an accelerator facility and the devices therein (vaccum part, electronic parts, etc.) currently I have about 1000 entities (nodes) in my graph where I am using the force atlas layout to put the nodes in position. The layouting is done on client side and takes - depending on the machine - between 10 seconds up to 30 seconds.
To speed up the layouting process I thought it might be useful to call the web page once on the server using PhantomJs (in a dark process), process the network (determine the end-positions of the nodes) and store the result as a JSON on the server which is called in the standard client request.

Do you see any ways to realize this idea with the given framework?

Thanks for any advice
WolfiG

**Comment by josdejong** [comment]
Yes you can save the final positions, check out the methods `storePositions()` and `getPositions()`
